### PR TITLE
Failing tests that cause timeouts when there is invalid group listings

### PR DESCRIPTION
**Pull request by kjetilk**
Please see the following tests. When I run them, all the Group ACL integration tests time out. 

The code seems to do a lot of loops through ```loadFrom```, but I haven't understood that at any depth. Perhaps that could be the reason for the timeout?

Anyway, please review these tests to see if they are tests that should pass. If they do make sense, it seems like the code is fragile at that point.

### CODE REVIEW

**Comment by RubenVerborgh** [comment]
If we want the build to succeed, we can write the tests using `it.skip`.

File: test/integration/acl-oidc-test.js [hunk 1]
```diff
@@ -578,6 +578,26 @@ describe('ACL with WebID+OIDC over HTTP', function () {
         done()
       })
     })
+    it('We should have a 500 with invalid group listings',
+      function (done) {
+        var options = createOptions('/group/test-folder/some-other-file.txt', 'user2')
+
+        request.get(options, function (error, response, body) {
+          assert.equal(error, null)
+          assert.equal(response.statusCode, 500)
```
[the remark this answers is not in the document]
Comment (kjetilk) [reply test/integration/acl-oidc-test.js]: What I think that this test is doing, I might be wrong, is that we're getting a certain file as a certain user, but the ACL for that resource is broken. That's why I think a 500 is appropriate. 

Obviously, the ACL should have been checked for validity when it was written (perhaps it is, I don't know), but that is beside the point, I think

**Comment by kjetilk** [comment]
Indeed, but the point is that if the tests are legit, they point out a bug.

File: test/integration/acl-oidc-test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (RubenVerborgh) [reply test/integration/acl-oidc-test.js]: 500 due to server misconfiguration makes sense indeed then.

**Comment by RubenVerborgh** [comment]
Yeah, I just meant that we might want to keep Travis green process-wise, and then strive to reduce the number of skipped tests (which is reported along with the number of passes and fails).

File: test/integration/acl-oidc-test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kjetilk) [reply test/integration/acl-oidc-test.js]: OK, we're in agreement. :-)

**Comment by kjetilk** [comment]
Certainly. So, the tests seem legit to you?

I can certainly make them skipped (there's no difference between skipped and todo tests?), so the question is if we should merge them and target fixing the bug that this may be causing for 5.0.0?

**Comment by RubenVerborgh** [comment]
Yes indeed. No difference between skipped and todo (but skipped implies todo for me).

Okay for merging with skipped.

**Comment by kjetilk** [comment]
OK, great stuff, I've done that and will open a new PR against develop shortly.

Just as an aside, the way I think about skipped vs TODO tests are that skipped tests are skipped due to some property of the test system, like the test relies of a certain OS feature, or of an optional module that may not be installed. TODO tests are for things that don't work yet. Skipped tests shouldn't run at all, whereas TODO tests should run to see if the issue has been fixed, but not reported as a failed test if they fail.

**Comment by RubenVerborgh** [comment]
Why a new PR? We can just continue this one :-)

**Comment by kjetilk** [comment]
Yeah, the PR was against the wrong branch, so I figured, I had to do it anyway, but perhaps I was wrong...

**Comment by RubenVerborgh** [comment]
You can change the target branch of PR.

**Pull request closed without merging**